### PR TITLE
rename Numpy -> NumPy

### PR DESCRIPTION
**Pull request by btel**
Following the PR for the numpy docs (https://github.com/numpy/numpy/pull/8021) I also suggest to rename all instances of "Numpy" on the website to the official spelling "NumPy" (capital N and P).

### CODE REVIEW

**Comment by certik** [comment]
That looks good to me. @rgommers ?

File: NA-overview.rst [hunk 1]
```diff
@@ -677,7 +677,7 @@ contributing code?
 If we go this route of writing up a decision process which includes
 such a dispute resolution mechanism, I think the meat of it should
 be a roadmap that potential contributers and developers can follow
-to gain influence over numpy. Numpy development needs broad support
+to gain influence over numpy. NumPy development needs broad support
```
Comment (jaimefrio) [inline NA-overview.rst]: There's a `numpy` just before the period that should also be changed. And I have spotted a number of other occurrences of `numpy` that refer to the project, not the module, in this same file, e.g. line 650.

**Comment by btel** [comment]
Thanks! I fixed the occurrences of numpy referring to the project.

**Comment by rgommers** [comment]
LGTM, remaining ``numpy``s refer to the package name or are links. Thanks @btel

**Pull request merged**